### PR TITLE
Update filtering.mdx

### PR DESCRIPTION
**Pull request by julauxen**
updating figma link

[Deploy Preview](https://deploy-preview-3994--keen-mayer-a86c8b.netlify.app/)

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

### CODE REVIEW

File: aries-site/src/pages/templates/filtering.mdx [hunk 1]
```diff
@@ -22,7 +22,7 @@ For details on the components that make up the filtering experience, see [Data d
     'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/filtering/FilteringCards/mockData.js',
   ]}
   docs="https://v2.grommet.io/data"
-  figma="https://www.figma.com/file/EA0OZFdocKVcQ55c7Jainb/HPE-Filter-Templates?node-id=681%3A1430"
+  figma="https://www.figma.com/design/fupkQ4sOefx3qUVoV6WfJi/Template-Library?m=auto&node-id=163-77&t=r7Bf1k6QfmCKSMhJ-1"
```
Comment (britt6612) [inline aries-site/src/pages/templates/filtering.mdx:25]: is this the correct link this takes me to form page? @julauxen

Comment (julauxen) [reply aries-site/src/pages/templates/filtering.mdx:25]: Hey @britt6612, yes, it's the only place we currently have filter examples. We might need to update this later to a more detailed template.

Comment (britt6612) [reply aries-site/src/pages/templates/filtering.mdx:25]: hmm I feel like the filtering example in the form is not the right filtering. That one is filtering validation on the form I think what we are looking for is a filtering in data and friends example. Should we just remove link?

Comment (julauxen) [reply aries-site/src/pages/templates/filtering.mdx:25]: hey @britt6612, we also have an example without validation that looks pretty close to the one on the website on that same link: 
Screenshot 2024-07-11 at 10 21 41

Comment (britt6612) [reply aries-site/src/pages/templates/filtering.mdx:25]: aw okay yes thats fine I missed this!

**Review by britt6612 — APPROVED** [review]

**Pull request merged**